redbot/core/cli.py

```diff
@@ -62,15 +62,16 @@ def ask_sentry(red: Red):
 
 
 def parse_cli_flags(args):
-    parser = argparse.ArgumentParser(description="Red - Discord Bot")
+    parser = argparse.ArgumentParser(description="Red - Discord Bot",
+                                     usage="redbot instance [arguments]")
```

Comment:
Just a tiny thing, I'd make the usage `redbot <instance_name> [arguments]`, so it lines up with the command help's positional arguments, as shown, this is how it is currently:
```
$ redbot --help
usage: redbot instance [arguments]

Red - Discord Bot

positional arguments:
  instance_name         Name of the bot instance created during `redbot-
                        setup`.
```

Reply:
Ye thats fair enough, will change it when I have some time.